Topic 1/src/com/globant/topic1/exercise1/model/Item.java

```diff
@@ -0,0 +1,66 @@
+package com.globant.topic1.exercise1.model;
+
+// Class that represent a cart item
+public class Item {
+	
+	// Fields
+	private Product product;
+	private int quantity;
+	
```

Comment:
a good idea for this kind of classes, is to have the Timestamp, something like
`LocalDate date = LocalDate.now();`
Also, if you check the equals method, it could be simply hacked by another sell with the same product and the same quantity. would be equals but actually they aren't. So with the timestamp we can avoid that.